github/ci/prow/files/jobs/kubevirt/kubevirt/kubevirt-periodics.yaml

```diff
@@ -164,13 +164,16 @@ periodics:
           export DOCKER_TAG="$(date +%Y%m%d)_$(git show -s --format=%h)" &&
           make &&
           make push &&
+          make build-functests &&
           git show -s --format=%H > _out/commit &&
           build_date="$(date +%Y%m%d)" &&
           echo ${build_date} > _out/build_date &&
           bucket_dir="kubevirt-prow/devel/nightly/release/kubevirt/kubevirt/${build_date}" &&
           gsutil cp ./_out/manifests/release/kubevirt-operator.yaml gs://$bucket_dir/kubevirt-operator.yaml &&
           gsutil cp ./_out/manifests/release/kubevirt-cr.yaml gs://$bucket_dir/kubevirt-cr.yaml &&
           gsutil cp -r ./_out/manifests/testing gs://$bucket_dir/ &&
+          gsutil cp -r ./_out/tests/tests.test gs://$bucket_dir/testing/ &&
+          gsutil cp -r ./_out/cmd/dump/dump gs://$bucket_dir/testing/ &&
```

Comment:
Do you need the dump binary? For reporting on test cases it is compiled in.

Reply:
I don't get it. How do I use the dump binary? Is it part of the tests.test?

Reply:
I see for example that there is a `dump` make target, which is called in `automation/test.sh`?

Reply:
And yes, I need it in case KubeVirt does not come up. In that case I want to print cluster state (same as it's done in automation/test.sh)

Reply:
> And yes, I need it in case KubeVirt does not come up. In that case I want to print cluster state (same as it's done in automation/test.sh)

That was what I needed to know. In this case you would need it.

> I don't get it. How do I use the dump binary? Is it part of the tests.test?

The dump binary is a think wrapper around the ginkgo-k8s-reporter which we have. So the test binary does not need it. That is what I meant.

Reply:
> And yes, I need it in case KubeVirt does not come up. In that case I want to print cluster state (same as it's done in automation/test.sh)

Oh, but openshift dumps the cluster-state for us alrady in this case, right?

Reply:
Hm, I think that would be a sane thing to do, not sure about that though. I can't remember if I added it just to mimic our build process or if the dump wasn't there. But I think it doesn't hurt, does it?

Reply:
@rmohr yes, openshift-ci dumps the cluster state for us already.

Reply:
OK, then. I'll remove the dump binary.